build.gradle

```diff
@@ -25,6 +25,7 @@ dependencies {
     annotationProcessor "org.projectlombok:lombok:1.18.12"
 
     testImplementation "org.junit.jupiter:junit-jupiter-api:5.5.1"
+    testImplementation "org.junit.jupiter:junit-jupiter-params:5.5.1"
```

Comment:
I add this dependency to be able to write parameterized tests with JUnit 5. If you don't agree with that, I can remove and duplicate all tests in GraphQLRequestSerializerTest.